ci/ci.sh

```diff
@@ -4,14 +4,26 @@ echo "INFO: Run id: $RUN_ID"
 echo "INFO: Sleep seconds set for services to start: ${SLEEP:=60}"  # defaults to 60 seconds
 
 if [[ $NODE_ENV == "local_client-public" ]]; then
-  $(node node_modules/@ceramicnetwork/cli/bin/ceramic daemon --verbose --log-to-files --network dev-unstable --anchor-service-api https://cas-qa.3boxlabs.com --ethereum-rpc https://rinkeby.infura.io/v3/b6685df41e1647c4be0046dfa62a020b) &
+  $(node node_modules/@ceramicnetwork/cli/bin/ceramic daemon --verbose --log-to-files --network dev-unstable --anchor-service-api https://cas-qa.3boxlabs.com --ethereum-rpc https://goerli.infura.io/v3/b6685df41e1647c4be0046dfa62a020b) &
 fi
 
 if [[ $NODE_ENV == "local_node-private" ]]; then
-  export IPFS_API_PORT=5011
-  export CERAMIC_NETWORK='dev-unstable'
-  export DEBUG='ipfs*error'
-  $(node node_modules/@ceramicnetwork/ipfs-daemon/bin/ipfs-daemon) &
+  # init config and generate peer id 
+  node_modules/go-ipfs/go-ipfs/ipfs init
+
+  # config changes to match js-ceramic/ipfs-daemon
+  node_modules/go-ipfs/go-ipfs/ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin "[\"*\"]"
```

Comment:
Are all these config changes to ipfs necessary, or was this just left over from trying a bunch of different things when the tests weren't passing?  I don't think this should be necessary and if it isn't it'd be best to use the default config to match what most of our users will be doing.

Reply:
These config changes were made to match the ones set in the ipfs-daemon package. I believe the only one that can go is this one: 
`node_modules/go-ipfs/go-ipfs/ipfs config --json Bootstrap "[\"/dns4/go-ipfs-ceramic-public-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmPP3RdaSWDkhcxZReGo591FWanLw9ucvgmUZhtSLt9t6D\", \"/dns4/go-ipfs-ceramic-public-qa-external.ceramic.network/tcp/4011/ws/p2p/QmUSSp4CY3wBALoy71T7BU4WjP3x9L5JzDJZkEDALmxhCq\", \"/dns4/go-ipfs-ceramic-private-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmXcmXfLkkaGbQdj98cgGvHr5gkwJp4r79j9xbJajsoYHr\", \"/dns4/go-ipfs-ceramic-private-cas-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmRvJ4HX4N6H26NgtqjoJEUyaDyDRUhGESP1aoyCJE1X1b\"]"`
@gvelez17 can you remove this?

Reply:
> These config changes were made to match the ones set in the ipfs-daemon package

Do we actually know if these changes are necessary?  We don't currently tell users that they can't just use off-the-shelf go-ipfs.  If there's something we're doing in our ipfs-daemon package that is **required** for the system to work, it would be good to know that

Reply:
I have used off-the-shelf go-ipfs locally but only for minor manual testing. I am not sure why these are set in the ipfs-daemon package and I am also not sure who to ask. When I was working on the go-ipfs shift I tried to leave the configs as they were. 

Here we used to run an ipfs node through our ipfs-daemon package so I tried to preserve the same settings. But if that is not important I don't mind removing a lot of these config changes

Reply:
the old ipfs-daemon package was primary for launching a js-ipfs node.  I'm not sure how necessary it is anymore for launching a go-ipfs node.  I'd expect an out-of-the-box go-ipfs node to work fine.  Maybe @smrz2001 or @v-stickykeys has some insight here.

I'd also be fine just merging this PR, then doing a follow-up one to remove the config settings and just see if it passes or not

Reply:
agreed.  will get this merged and then do a separate pr to test removing those